src/library/StatBoxList/Timeleft.tsx

```diff
@@ -23,7 +22,19 @@ export const Timeleft = ({
     <StatBox>
       <div className="content chart">
         <div className="chart">
-          <StatPie value={graph?.value1} value2={graph?.value2} />
+          <Chart
+            items={[
+              {
+                value: graph?.value1,
+                color: 'var(--accent-color-primary)',
+              },
+              {
+                value: graph?.value2,
+                color: 'var(--background-default)',
+              },
+            ]}
+            diameter={30}
```

Comment:
Matching the size of current solution.
```suggestion
            diameter={34}
```